build.sh

```diff
@@ -40,14 +41,14 @@ prepare_bootstrap()
 un_prepare_bootstrap()
 {
     if [ -f "${ROOTFS_DIR}${QEMU_STATIC_BIN}" ]; then
-        rm  -f "${ROOTFS_DIR}${QEMU_STATIC_BIN}"
+        "${RUN_SUDO}" rm  -f "${ROOTFS_DIR}${QEMU_STATIC_BIN}"
```

Comment:
I'm wondering, why do we need sudo to remove the file, but not when copying it to the location?
I suspect copy operation will place the file in the rootfs (as $USER) and thus we should be able to delete it again as well? It is our file after all.